deenurp/subcommands/rdp_extract_genbank.py

```diff
@@ -70,8 +83,17 @@ def is_type(record):
     return '(T)' in record.description
```

Comment:
might as well just update this function to include search terms for type strains